### PR TITLE
CB Response: Add child process max

### PR DESCRIPTION
**Pull request by avanbrunt-cb**

### CODE REVIEW

File: src/cbapi/response/models.py [hunk 1]
```diff
@@ -1407,6 +1407,30 @@ def group_by(self, field_name):
...
             return self
 
+    def max_children(self, num_children):
+        """Sets the number of children to fetch with the process
+
+        This method is only available for Cb Response servers 6.0 and above. Calling this on a Query object connected
+        to a Cb Response 5.x server will simply result in a no-op.
+
+        :default: 15
+        :param int num_children: Number of children to fetch with process
```
Comment (abowersox-cb) [inline src/cbapi/response/models.py]: Does this service have a means of specifying "no limit on number of children"? Such as by setting max_children to 0 or a negative number? If so, it ought to be documented.

Comment (avanbrunt-cb) [reply src/cbapi/response/models.py]: Nope but that is a good point I should check for negative

Comment (avanbrunt-cb) [reply src/cbapi/response/models.py]: Never mind the API handles that for me it will just return zero results if negative or zero

Comment (abowersox-cb) [reply src/cbapi/response/models.py]: Pity it doesn't have an override to remove all limits. Oh well, I guess you can just pass in 99999 or some suitably high number if you want to have an effective "no limit."

**Pull request merged**